scripts/azure-pipelines/test-modified-ports.ps1

```
@@ -165,9 +165,7 @@ if (($BuildReason -eq 'PullRequest') -and -not $NoParentHashes)
     $parentHashesFile = Join-Path $WorkingRoot 'parent-hashes.json'
     $parentHashes = @("--parent-hashes=$parentHashesFile")
     & git revert -n -m 1 HEAD | Out-Null
-    & "./vcpkg$executableExtension" ci $Triplet --dry-run --exclude=$skipList @hostArgs @commonArgs --no-binarycaching "--output-hashes=$parentHashesFile" `
-        | ForEach-Object { if ($_ -match ' dependency information| determine pass') { Write-Host $_ } }

```

Did this hide error output? Do we need the full standard output?

---

Yes it did. Triggered the msi issue and that resulted in lots of confusion